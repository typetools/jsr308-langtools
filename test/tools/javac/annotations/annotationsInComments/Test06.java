/*
 * @test
 * @bug 1234567
 * @summary Annotations in comments are recognized
 * @author Werner Dietl
 * @compile/fail/ref=Test06.out -XDrawDiagnostics -XDTA:spacesincomments Test06.java
 */

// Recognized comment, unknown annotation
/* @Unknown("HiThere!") */
class Test06 {}
