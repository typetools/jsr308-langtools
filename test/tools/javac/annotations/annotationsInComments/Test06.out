Test06.java:10:5: compiler.err.cant.resolve: kindname.class, Unknown, , 
1 error